apps/preview/next/pages/showcases/RatingForms/ProductRatingWithReview.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useState, useId } from 'react';
+import classNames from 'classnames';
+import { SfButton, SfRatingButton, SfInput } from '@storefront-ui/react';
+
+export default function ProductRatingWithReview() {
+  const [rating, setRating] = useState(0);
+  const [review, setReview] = useState('');
+  const [username, setUsername] = useState('');
+  const [reviewCharacterLimit] = useState(5000);
+  const ratingLabelId = useId();
+
+  const reviewIsAboveLimit = review.length > reviewCharacterLimit;
+  const reviewCharsCount = reviewCharacterLimit - review.length;
+
+  return (
+    <div className="max-w-[376px] md:max-w-[768px]">
+      <h3 className="font-bold py-2 pl-4 pr-3 typography-headline-4">Leave a review</h3>
+      <form className="grid grid-cols-[100px_1fr] py-2 px-4 gap-4 md:grid-cols-[176px_1fr] grid-rows-[100px_1fr] md:grid-rows-[28px_1fr] items-center md:items-start">
+        <img
+          src="http://localhost:3100/@assets/smartwatch.png"
+          alt="Smartwatch"
+          width="100"
+          height="100"
+          className="mx-auto border border-neutral-200 rounded-md aspect-square w-[100px] md:w-[176px]"
+        />
```

Comment:
question: is width attribute still needed here `width="100"` when you have `w-[100px]` class?

Reply:
I think it's better to have them as some tools like lighthouse check if they are added